src/modules/authentication/AuthenticationWrapper.ts

```diff
@@ -76,7 +76,7 @@ export class AuthenticationWrapper implements IAuthenticationWrapper {
     }
   }
 
-  private getAccount(): AccountInfo | undefined {
+  public getAccount(): AccountInfo | undefined {
```

Comment:
reminder to revert this to 'private'

Reply:
turns out I need this function in the feedback component